docs/tutorials/repository-tutorial/01-modeling-and-features.rst

```diff
@@ -0,0 +1,41 @@
+Introduction to Database Modeling and Repository Features
+---------------------------------------------------------
+In this tutorial, we will cover the integrated repository features in Litestar 2.0, starting with database modeling using the included SQLAlchemy declarative model helpers. These are a series of classes and mixins that incorporate commonly used functions/column types to make working with models easier.
+
+.. literalinclude:: /examples/contrib/sqlalchemy/sqlalchemy_declarative_models.py
```

Comment:
This file isn't in the change-set, so I'll just note here that the plugin config created in the example unnecessarily specifies the `session_dependency_key` argument:

```py
sqlalchemy_config = SQLAlchemyAsyncConfig(
    connection_string="sqlite+aiosqlite:///test.sqlite", session_dependency_key="db_session"
)
```
`"db_session"` is the default ([ref](https://github.com/litestar-org/litestar/blob/c0a98e51ee09efb15f5892e6143d0a6fa5fd7cd9/litestar/contrib/sqlalchemy/plugins/init/config/common.py#LL115C1-L115C1))